### PR TITLE
Feat: Typography, Style: Typography styling

### PR DESCRIPTION
**Pull request by Rishi05051997**
Please review typography

### CODE REVIEW

File: components/Typography/typography.html [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+            <li >pad-xs</li>
+            <li >pad-sm</li>
+            <li >pad-md</li>
+            <li >pad-lg</li>
+            <li >pad-xl</li>
+            <li >pad-xxl</li>
+            <li >pad-infinite(10rem)</li>
+          </ul>
+
+
```
Comment (devdeadviz) [inline components/Typography/typography.html]: remove these spaces

File: components/Typography/typography.html [hunk 2]
```diff
@@ -0,0 +1,370 @@
...
+            <li >pad-y-1 <span  class="imp-highlight-text">xs</span></li>
+            <li >pad-y-2 <span  class="imp-highlight-text">sm</span></li>
+            <li >pad-y-3 <span  class="imp-highlight-text">md</span></li>
+            <li >pad-y-4 <span  class="imp-highlight-text">lg</span></li>
+            <li >pad-y-5 <span  class="imp-highlight-text">xl</span></li>
+            <li >pad-y-6 <span  class="imp-highlight-text">xxl</span></li>
+          </ul>
+
+          
+
```
Comment (devdeadviz) [inline components/Typography/typography.html]: same here

File: components/Typography/typography.html [hunk 3]
```diff
@@ -0,0 +1,370 @@
...
+            <li >mar-xs</li>
+            <li >mar-sm</li>
+            <li >mar-md</li>
+            <li >mar-lg</li>
+            <li >mar-xl</li>
+            <li >mar-xxl</li>
+          </ul>
+
+          
+
```
Comment (devdeadviz) [inline components/Typography/typography.html]: same here

File: components/Typography/typography.html [hunk 4]
```diff
@@ -0,0 +1,370 @@
...
+              sandbox="allow-scripts allow-same-origin">
+            </iframe>
+          </div>
+
+          <div class="text-4 mar-y-4">Border Radius Utilities</div>
+          <p class="text-3">Border Radius comes with small <span class="imp-highlight-text">(0.5rem)</span>, mediums <span class="imp-highlight-text">(1rem)</span>, large <span class="imp-highlight-text">(1.5rem)</span> and extra large <span class="imp-highlight-text">(50%)</span>
+          </p>
+          <div class="img_container sm-bdr-radius mar-y-2 pad-md">
+            small Border Radius
+          </div>
```
Comment (devdeadviz) [inline components/Typography/typography.html]: fix indentation

File: components/Typography/typography.html [hunk 5]
```diff
@@ -0,0 +1,370 @@
...
+
+          <div class="img_container xl-bdr-radius mar-y-2 pad-md">
+            extra large Border Radius
+          </div>
+          
+          <div class="img_container pad-md mar-y-2">
+            <iframe
+              src="https://carbon.now.sh/embed/cxzjR9sAdfoFhsb2x9ev"
+              style="width: 308px; height: 169px; border:0; transform: scale(1); overflow:hidden;"
+              sandbox="allow-scripts allow-same-origin">
```
Comment (devdeadviz) [inline components/Typography/typography.html]: fix indentation

**Review by devdeadviz — CHANGES_REQUESTED** [review]
added some changes

File: components/Typography/typography.html (hunk 1 above)
Comment (Rishi05051997) [reply components/Typography/typography.html]: Ok removing spaces

File: components/Typography/typography.html (hunk 4 above)
Comment (Rishi05051997) [reply components/Typography/typography.html]: Yes doing that

File: components/Typography/typography.html (hunk 5 above)
Comment (Rishi05051997) [reply components/Typography/typography.html]: Completed indentation issue

File: components/Typography/typography.html (hunk 3 above)
Comment (Rishi05051997) [reply components/Typography/typography.html]: Removed Spaces

**Pull request merged**